### ISSUE TITLE
proposal for proportional payments to entities, based on a fraction of a set payment at set intervals

### ISSUE THREAD

**Issue by ericspivey** [liberapay/liberapay.com#2369]
TLDR: proposal for proportional payments to entities, based on a fraction of a set payment at set intervals.

I have thought a lot about why more people don't support free software. You are already doing a lot at Liberapay to change this, but I think there could be things done that would make giving more streamlined. I say this as someone who "supports" free software, but has not paid for it yet because it is not easy enough for something that I don't "have" to do.

I think about the barriers that I have had and/or most people who are not giving probably have, and they are:
- ease of payment method (Liberapay mostly addresses this). I first encountered Liberapay on F-Droid. I saw the icon for making a payment with Liberapay, but assumed I would not be able to use a credit card. You may want to consider adding Visa/MC (or other region-specific) icons to that button.
- security of payment (Liberapay addresses this)
- integrity of payment (Liberapay addresses this) Kudos to y'all for making everything so transparent!
- payment amount/recurrance. There are a lot of problems here from a vain/lazy person's perspective (which is most of us, at least some of the time). This is where you could make thigs a lot easier. First, a lot of people would rather give nothing than look like a cheapskate (guilty). Second, if you have to think about how much to give *one* entity, you have to think about how much you're NOT giving other entities you would maybe like to, and then you think how much you would be paying if you were, and that ends up being more money/effort than you want, so you end up doing nothing.

Here's what I propose: proportional payments based on a fraction of a set payment at set intervals.
Here's the minimum the user would have to do:
- Set an amount and frequency they would be charged
	+ instead of Liberapay suggesting a total amount, suggest a fraction (5%?) of the user's disposable income
	+ default to monthly frequency on the current day of the month
- Start adding entities they want to pay
	+ you should automatically ask to add Liberapay
	+ the donation should be anonymous by default (so user doesn't look cheap)
THAT'S IT. If nothing else was done, every entity added would get an equal share of the total amount at the set frequency. This is great because it minimizes mental barriers to giving, and people can just get started doing it and "perfect" it later.

Additional things that should/could be done:
- Reverse compatibility. Users could still do everything they do now with Liberapay
- Optionally, a user could set multiple set amounts and frequencies, "bins", to which different types of entities could be added (eg., apps bin, arts bin, organizations bin, etc). They wouldn't have to decide that initially, though, there would just be a default initial bin.
- Optionally, users could increase the "shares" for any given entity in the bin for things that they want to support more. Example, if you had four entities in a bin, they would by default each get 25% of the total amount for that bin (equal split four ways). If you increased the shares of one entity you liked best to 3 shares, the other three would still have one share each for a total of 6 shares. The entity you liked best would have 3/6 or 50% of the bin total amount, and the others would each have 1/6 or 16% of the total.
- You could prompt the user to boost the total bin payment (or reduce shares of high-share entity) when the lowest donation approaches an insignificant sum.
- MAKE AN ANDROID APP ON F-DROID. Have this app start by default with F-Droid and Liberapay as two entities in an "App" bin. Give users the option of adding their F-Droid installed applications to the bin automatically.
